site/content/docs/user/quick-start.md

```diff
@@ -366,8 +366,7 @@ kubeadmConfigPatches:
 {{< /codeFromInline >}}
 
 #### IPv6 clusters
-You can run ipv6 only clusters using `kind`, but first you need to enable ipv6 in your docker daemon by editing `/etc/docker/daemon.json` [as described here][docker enable ipv6].
-Ensure you `systemctl restart docker` to pick up the changes.
+You can run ipv6 only clusters using `kind`, but first you need to check that ou have enabled ipv6 in your host.
```

Comment:
I don't think this comment is complete, for one thing it's the *docker* host (people do use remote docker) and second there's docker desktop.

Reply:
right